### PR TITLE
add poo status to institutions and institutions_archives tables

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
Database migration PR for https://github.com/department-of-veterans-affairs/va.gov-team/issues/10358

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

File: db/migrate/20200626165535_add_poo_status_to_institutions.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class AddPooStatusToInstitutions < ActiveRecord::Migration[5.2]
+  def change
+    add_column :institutions, :poo_status, :string
```
Comment (kfrz) [inline db/migrate/20200626165535_add_poo_status_to_institutions.rb:3]: Just confirming - no default value needed?

**Review by kfrz — APPROVED** [review]
LGTM, just a question re: default values?

File: db/migrate/20200626165535_add_poo_status_to_institutions.rb (hunk 1 above)
Comment (zurbergram) [reply db/migrate/20200626165535_add_poo_status_to_institutions.rb:3]: correct, the value should be provided in the WEAMS csv and is inserted into institutions during use of https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/app/models/weam.rb#L20 which is updated in https://github.com/department-of-veterans-affairs/gibct-data-service/pull/676

**Pull request merged**